xml/gl.xml

```diff
@@ -26900,7 +26900,7 @@ typedef unsigned int GLhandleARB;
         </command>
         <command>
             <proto>void <name>glVertexArrayParameteriAPPLE</name></proto>
-            <param group="VertexArrayPNameAPPLE"><ptype>GLenum</ptype> <name>pname</name></param>
+            <param group="VertexArrayParameterNameAPPLE"><ptype>GLenum</ptype> <name>pname</name></param>
```

Comment:
Yeah, I didn't remove vendor suffixes when possible in this PR. However, it might be better to remove them, depending on the discussion in #481.

The reason for renaming is that `VertexArrayPName` is already a group. And a separate one, with its own uses and enum values.
And tbh I'm sure there was a reason I expanded `P` into `Parameter`, I think something to do with functions in which this group is used. But I can't find it now.
Well, I'm all ears for naming suggestions, as always, because naming sense is not my forte.

Regardless, this PR is exactly about resolving such collisions between groups, be it by merging or splitting by renaming.
Of course, every group is its own case, but I think making a swarm of PR is a bad idea.
That said, I can make 1 PR for all the group merges and 1 PR for all splits by renaming. Do you think that would be better?